cinema/serializers.py

```diff
@@ -125,9 +125,9 @@
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
```

Comment:
Ensure that the `Ticket.validate_ticket` method is correctly implemented and that it accepts the parameters being passed here. Also, verify that `ValidationError` is being used correctly within this context.